WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2492,6 +2496,24 @@ public void done() {
                                                          Activity.RESULT_OK, data);
                     }
                     break;
+                case RequestCodes.STOCK_MEDIA_PICKER:
+                    if (data != null && data.hasExtra(StockMediaPickerActivity.KEY_UPLOADED_MEDIA_IDS)) {
+                        long[] mediaIds =
+                                data.getLongArrayExtra(StockMediaPickerActivity.KEY_UPLOADED_MEDIA_IDS);
+
+                        Map<String, Object> properties = new HashMap<>();
+                        properties.put("count", mediaIds.length);
+                        AnalyticsUtils.trackWithSiteDetails(
+                                Stat.EDITOR_ADDED_PHOTO_VIA_STOCK_MEDIA_LIBRARY,
+                                mSite,
+                                properties);
+
+                        for (long id : mediaIds) {
+                            addExistingMediaToEditor(id);
```

Comment:
I noticed that `addExistingMediaToEditor` already bumps analytics when it calls `trackAddMediaFromWPLibraryEvents` (aka EDITOR_ADDED_VIDEO_VIA_WP_MEDIA_LIBRARY). 

Is it on purpose?

Reply:
> I noticed that addExistingMediaToEditor already bumps analytics when it calls trackAddMediaFromWPLibraryEvents

I totally missed that - thanks for catching it. I resolved this by adding a "source" parameter to the tracking method [here](https://github.com/wordpress-mobile/WordPress-Android/blob/975837e5f013fc6dda20d36ae7d7a2d615b9dd1a/WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java#L2214).

Reply:
👍